src/flask/run_flask_app.py

```diff
@@ -0,0 +1,13 @@
+from flask_injector import FlaskInjector
+
+from flask import Flask
+from src.core.di_config import DiProviders
+from src.flask.routes.chef import chef_blueprint
+
+app = Flask(__name__)
+app.register_blueprint(chef_blueprint)
```

Comment:
Wrap this into a setup function as well.

```
def setup _api(app):
```

Reply:
Call chef setup function here and get the ioc here to be send the chef setup function

Reply:
done.